libclc/CMakeLists.txt

```diff
@@ -297,6 +304,10 @@ foreach( t ${LIBCLC_TARGETS_TO_BUILD} )
     set( DARCH ${ARCH} )
   endif()
 
+  if( (ARCH  STREQUAL x86_64 OR ARCH  STREQUAL aarch64) AND OS STREQUAL "linux")
```

Comment:
What would happen if you try and build this on something other than `x86_64` or `aarch64`, or don't use `linux`? Would the build fail? Would it build silently and do the wrong thing? I think we should think about how to safeguard the build from unsupported native-cpu configurations.

Reply:
Also there are double spaces here, and inconsistent string quoting.

Reply:
This is a bit tricky, I've added a warning that is triggered when building libclc for a "native cpu arch" on an OS != `linux`. Other cases are a bit hard to detect: different triples may or may not be pre-existing DPC++ targets (`ptx`, `amdgcn` etc), and at this stage I think that (for Native CPU), we would like them to fall back to `generic` and see what happens